sequencer.c

```diff
@@ -2653,6 +2654,29 @@ int sequencer_rollback(struct repository *r, struct replay_opts *opts)
 	return -1;
 }
 
+int sequencer_skip(struct repository *r, struct replay_opts *opts)
+{
+	switch (opts->action) {
+	case REPLAY_REVERT:
+		if (!file_exists(git_path_revert_head(r)))
+			return error(_("no revert in progress"));
+		break;
+	case REPLAY_PICK:
+		if (!file_exists(git_path_cherry_pick_head(r)))
+			return error(_("no cherry-pick in progress"));
+		break;
+	default:
+		BUG("the control must not reach here.");
+	}
+
+	if (file_exists(git_path_seq_dir())) {
```

Comment:
Is this correct? `file_exists()` on a directory? Shouldn't this be `dir_exists()` instead?

Reply:
I wonder if there is a function `dir_exists`, everything is a file in Linux! No, seriously, I wasn't able to find `dir_exists`. Although there exists one for `clone.c` to use.

Reply:
cf. https://github.com/gitgitgadget/git/issues/230

Reply:
Yep, my bad. I should have looked before suggesting `dir_exists()`. The function I wanted to suggest is actually called `is_directory()`.